### DISCUSSION TITLE
Problem 3.5.0 AMD GPU

### DISCUSSION THREAD

**Discussion by DJRoby19** [mathoudebine/turing-smart-screen-python#530] [General]
Hello everyone, someone who uses version 3.5.0 on Linux with an AMD 7800xt video card, it happens to you too

In this version 3.5.0 the AMD 7800xt GPU is not detected on the Asus motherboard but the CPU fan works
however in version 3.4.0 it was the opposite
But I see the GPU degrees, the percentage remains at 0% and the fan speed is empty.
In the log file it tells me this
20/06/2024 18:18:36 [INFO] Loading theme Cyberdeck from res/themes/Cyberdeck/theme.yaml
20/06/2024 18:18:36 [DEBUG] HW revision: A
20/06/2024 18:18:36 [DEBUG] Auto detected COM port: /dev/ttyACM0
20/06/2024 18:18:36 [DEBUG] Using Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
20/06/2024 18:18:36 [WARNING] Tray icon is not supported on your platform
20/06/2024 18:18:36 [INFO] Display reset (COM port may change)...
20/06/2024 18:18:41 [DEBUG] Static COM port: /dev/ttyACM0
20/06/2024 18:18:42 [DEBUG] HW sub-revision: SubRevision.TURING_3_5
20/06/2024 18:18:42 [DEBUG] Drawing Image: BACKGROUND
20/06/2024 18:18:42 [DEBUG] Bitmap res/themes/Cyberdeck/background.png is now loaded in the cache
20/06/2024 18:18:42 [INFO] Detected AMD GPU(s)
20/06/2024 18:18:42 [WARNING] Your GPU FPS is not supported yet
20/06/2024 18:18:42 [WARNING] Your GPU Fan Speed is not supported yet

**Comment by DJRoby19** [comment]
Is it possible that I'm the only one having this problem under Linux?

**Reply by gigi798** [reply]
i have the same problem on windows

**Reply by DJRoby19** [reply]
Then I reported it as a bug but no one responded for a solution

**Reply by gigi798** [reply]
> Then I reported it as a bug but no one responded for a solution

i fixed my issue by replacing the librehardwaremonitor .dll files with https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/releases ones  they are 0.9.4.0.0  while this project use 0.9.3.0.0

**Reply by DJRoby19** [reply]
Thanks
